K2Bridge/JsonConverters/TermsBucketAggsConverter.cs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// See LICENSE file in the project root for full license information.
+
+namespace K2Bridge.JsonConverters
+{
+    using K2Bridge.JsonConverters.Base;
+    using K2Bridge.Models.Response;
+    using Newtonsoft.Json;
+
+    /// <summary>
+    /// A converter able to serialize Bucket aggregations to Elasticsearh response json format.
+    /// </summary>
+    internal class TermsBucketAggsConverter : WriteOnlyJsonConverter
```

Comment:
same exact at the histogram bucket converter?

Reply:
The Terms bucket does not have `key_as_string`... It's a small difference but I think the buckets will look a bit different for each bucket aggregation, with some fields added and other fields removed. And the converter casts the object to the right bucket type at the beginning, to access the properties.